roles/ansible-role-postgresql/tasks/configure-for-tdp-getting-started.yml

```diff
@@ -1,13 +1,4 @@
 ---
-- name: Trust connections from postgresql_client (add to pg_hba.conf)
-  blockinfile:
-    path: "{{ postgresql_data_dir }}/pg_hba.conf"
-    block: |
-      host all all {{ hostvars[item]['ansible_host'] }}/32 trust
-      host all all {{ item }}.{{ hostvars[item]['domain'] }} trust
-    marker: "# {mark} ANSIBLE MANAGED BLOCK {{ item }}"
-  loop: "{{ groups['postgresql_client'] }}"
-
 - name: Trust connections from the ansible host (add to pg_hba.conf)
   lineinfile:
     dest: /var/lib/pgsql/data/pg_hba.conf
```

Comment:
We could arguably totally remove `configure-for-tdp-getting-started.yml`. Do we really need to reach postgres from the Ansible host ?

Reply:
Yes we could remove that. I don't see any need to reach PostgreSQL from the Ansible host. But I also think that this role should be entirely recreated in the refactor of the prerequisites to I didn't put much effort to make it clean in this PR.